.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,15 @@
+### Description
+
```

Comment:
I would just remove this empty line between the title and the description to keep the standard